content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,30 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- [`cy.origin()`](/api/commands/origin) now supports using `require()` and dynamic `import()` to include
+  dependencies. `Cypress.require()` has been removed. Addresses
+  [#24293](https://github.com/cypress-io/cypress/issues/24293).
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
+Ensures that with `experimentalSessionAndOrigin` enabled, tests are still able to run against webkit. Addresses [#24116](https://github.com/cypress-io/cypress/issues/24116), [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses [#23379](https://github.com/cypress-io/cypress/issues/23379), [#23766](https://github.com/cypress-io/cypress/issues/23766), [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.
+  - Users will be prompted to record runs to the Dashboard from both the Cypress App and Cypress Launchpad if a project is connected to the Dashboard and has no recorded runs.
+
+**Bugfixes:**
+
+- Fixed issue connecting to the cloud when a self-signed cert was in the cert chain. Fixes [#24298](https://github.com/cypress-io/cypress/issues/24298).
+- The way that Cypress handles [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation) has changed.  The previous modes of `legacy` and `strict` have been replaced with `on` and `off`.  Fixes [#24206](https://github.com/cypress-io/cypress/issues/24206).
+  - The default mode is `on` when [`experimentalSessionAndOrigin`](/guides/references/experiments) is enabled.
+  - The [cy.session()](/api/commands/session) command now inherits the test isolation behavior for the suite it runs in.
+- The behavior and implementation of the experimental strict test isolation mode has been evaluated per user feedback. strict mode has been replaced test isolation on and off modes, with on mode being the default behavior observed when experimentalSessionAndOrigin=true and the cy.session() command now inherits the test isolation behavior for the suite it runs in. When enabled, legacy mode is not longer a valid mode and to persist the browser context between tests off mode can be used. Fixes [#24206](https://github.com/cypress-io/cypress/issues/24206).
```

Comment:
These both reference the same issue, should we combine them?

Reply:
Yes, I meant to take out the second one.  Will do now.